build/hellobuild.md

```diff
@@ -0,0 +1,136 @@
+<br/>Dockerfiles are crucial inputs for image builds and can facilitate automated, multi-layer image builds based on your unique configurations. Dockerfiles can start simple and grow with your needs and support images that require complex instructions.
+  
+For all the possible instructions, see the [Dockerfile reference](../engine/reference/builder/).
+
+## Example
+Here’s a simple Dockerfile example to get you started with building images. We’ll take a simple "Hello World" Python Flask application, and bundle it into a docker image that we can test locally or deploy anywhere!
+
+**Sample A**
+<br/>Let’s say we have the following in a `hello.py` file in our local directory:
+
+{% highlight python lineos %}
+from flask import Flask
+app = Flask(__name__)
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+{% endhighlight %}
+
+Don’t worry about understanding the full example if you’re not familiar with Python - it’s just a simple web server that will contain a single page that says “Hello World”.
+
+> **Note:** If you test the example, make sure to copy over the indentation as well!
+
+**Sample B**
+<br/>Here’s a Dockerfile that Docker Build can use to create an image for our application:
+
+{% highlight dockerfile lineos %}
+# syntax=docker/dockerfile:1
+FROM ubuntu:22.04
+
+# install app dependencies
+RUN apt-get update && apt-get install -y python3 python3-pip
+RUN pip install flask
+
+# install app
+COPY hello.py /
+
+# final configuration
+ENV FLASK_APP=hello
+    EXPOSE 8000
+CMD flask run --host 0.0.0.0 --port 8000
+{% endhighlight %}
+
+* `# syntax=docker/dockerfile:1` 
+
+    This is our syntax directive. It pins the exact version of the dockerfile syntax we’re using. As a [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/), this should be the very first line in all our Dockerfiles as it informs Buildkit the right version of the Dockerfile to use. 
+    <br/>See also [Syntax](../engine/reference/builder/#syntax).
+    
+    > **Note:** Initiated by a "#" like regular comments, this line is treated as a directive when you are using BuildKit (default), otherwise it is ignored.
+
+
+* `FROM ubuntu:22.04`
+
+    Here the `FROM` instruction sets our base image to the 22.04 release of Ubuntu. All following instructions are executed on this base image, in this case, a Ubuntu environment. 
+    The notation `ubuntu:22:04`, follows the `name:tag` standard for naming docker images. 
+    When you build your image you use this notation to name your images and use it to specify any existing docker image.
+    There are many public images you can leverage in your projects. <br/>Explore [Docker Hub](https://hub.docker.com/_/ubuntu/?tab=tags) to find out.
```

Comment:
```suggestion
    There are many public images you can leverage in your projects. Explore [Docker Hub](https://hub.docker.com/_/ubuntu/?tab=tags) to find out.
```

Reply:
Maybe we should just link :link: to the main docker hub home page here, or some other list of official images like `https://hub.docker.com/search?image_filter=official&q=&type=image`, instead of just the Ubuntu images.